### DISCUSSION TITLE
[@types/plotly.js] Config is missing autotypenumbers option

### DISCUSSION THREAD

**Discussion by eddeisling** [DefinitelyTyped/DefinitelyTyped#64197] [Issues with a @types package]
Current definitions do not include the **autotypenumbers** option. It should be presented in layout and in xaxis/yaxis
https://github.com/plotly/plotly.js/blob/b54a3dcdfe5a4a808af26473b516d40526fbf2f9/src/plots/cartesian/layout_attributes.js#L261

```typescript
autotypenumbers: 'convert types' | 'strict'
```

[a comment by an automated account is not shown]

**Comment by NuclearPhoenixx** [comment]
Yeah, many other definitions are missing too, like layout.xaxis.rangeslider.autorange or layout.newselection. Generally anything newer than 2.12 I suspect. 😢
